src/calendar/day/period/index.tsx

```diff
@@ -128,9 +128,9 @@ const PeriodDay = (props: PeriodDayProps) => {
     const end = markingStyle.endingDay;
 
     if (start && !end) {
-      rightFillerStyle.backgroundColor = markingStyle.startingDay?.backgroundColor;
+      rightFillerStyle.backgroundColor = marking?.fillerColor ?? markingStyle.startingDay?.backgroundColor;
```

Comment:
this is not the place I would check for the attribute. A better one is in the definition of `markingStyle` where we set the `.startingDay` and `.endingDay` (lines #51 and 54). For example:
```
 defaultStyle.startingDay = {backgroundColor: marking.fillerColor ?? marking.color};